internal/handler/wallet/delete.go

```diff
@@ -6,11 +6,20 @@ import (
 	"github.com/massalabs/thyra-plugin-wallet/api/server/models"
 	"github.com/massalabs/thyra-plugin-wallet/api/server/restapi/operations"
 
+	"github.com/massalabs/thyra-plugin-wallet/pkg/delete"
 	"github.com/massalabs/thyra-plugin-wallet/pkg/wallet"
 )
 
+func NewDelete(deleteConfirm delete.Confirmer) operations.RestWalletDeleteHandler {
+	return &walletDelete{deleteConfirm: deleteConfirm}
+}
+
+type walletDelete struct {
+	deleteConfirm delete.Confirmer
```

Comment:
This field is poorly named. It doesnt bring any more insight than its type on what it is.